packages/core/utils/src/types.ts

```diff
@@ -1,5 +1,5 @@
-import * as Koa from 'koa';
-import 'koa-body';
+import type * as Koa from 'koa';
+import type {} from 'koa-body';
```

Comment:
What is that for?

Reply:
I need the side effects of the types (extends the koa body)